src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantSubmitAppealPersonalisationEmail.java

```diff
@@ -59,13 +61,18 @@ public String getReferenceId(Long caseId) {
     @Override
     public Map<String, String> getPersonalisation(AsylumCase asylumCase) {
         requireNonNull(asylumCase, "asylumCase must not be null");
+
+        final String dueDate =
+            LocalDate.now().plusDays(14)
+                .format(DateTimeFormatter.ofPattern("d MMM yyyy"));
```

Comment:
Can we pull the date formatter to a constant that we can reuse in other places then we just should show all dates in the same format

Reply:
Thanks for pointing this out. I've applied the suggestion and moved this to a re-usable method in the SystemDateProvider